### DISCUSSION TITLE
No maintainer for semantic-release/changelog?

### DISCUSSION THREAD

**Discussion by boxcee** [semantic-release/semantic-release#2263] [Q&A] [answered]
## Current behavior

Non-code issue. I opened a PR with `semantic-release/changelog`, but didn't get any feedback from a maintainer for almost a year now. Got feedback from some contributors.

## Expected behavior

Get feedback on open PR's.

**Comment by travi** [comment, marked as answer]
the maintainers of semantic-release actively maintain all of the projects in this organization. we are a small team and work on the project in our spare time. unfortunately, we are unable to keep up with all of the activity, which means that some contributions lack our attention. your PR has not been completely ignored. we are aware of it, but have not been able to respond. we are trying very hard to keep everything moving forward in a valuable direction, but some things have not gotten our attention.

please understand that your PR was opened without any [agreement from the core team](https://github.com/semantic-release/semantic-release/blob/master/CONTRIBUTING.md#feature-request) that it is functionality that would be accepted. we have not closed the PR because we have not ruled out the possibility of including the capability, but we have to consider the complexity it adds to the project against the percentage of our large user base that would benefit from the addition. while we appreciate the effort you've put into your PR, that alone is not enough to expect PR acceptance. 

with all of that said, the changelog plugin is open source software. if there is a change that would be valuable to you that is not moving quickly in the official plugin, you are welcome to fork the project and publish your own version under your own scope. often, we will ask for this approach before accepting a contribution anyway, in order to prove out the behavior before releasing to all of our users. if you do so, please link to it in the related issue or PR thread so that others can find it and we can understand demand from the community if we do give it attention in the future.

**Reply by boxcee** [reply]
Thanks a lot for this answer. Appreciate it!
